keras/src/backend/tensorflow/random_test.py

```diff
@@ -0,0 +1,20 @@
+import tensorflow as tf
+
+from keras.src.backend.tensorflow import random
+from keras.src.testing import TestCase
+
+
+class TFRandomTest(TestCase):
```

Comment:
You'll have to mark this one as tensorflow only, it's failing with the torch backend.

Reply:
Done, thanks.